src/components/checkboxes/index.md.njk

```diff
@@ -0,0 +1,35 @@
+---
+title: Checkboxes
+description: Let users select one or more options by using the Checkboxes component.
+section: Components
+theme:
+status: Approved
+department: All
+aliases:
+copy: done
+layout: layout-pane.njk
+---
+
+Let users select one or more options by using the Checkboxes component.
+
+## When to use this component
+
+Use the Checkboxes component to help users:
+
+- select multiple options from a list
+- toggle a single option on or off
+
+## When not to use this component
+
+Don’t use the Checkboxes component if users can only choose one option from a selection. In this case, use the [Radios component](Insert URL to Radios component page).
```

Comment:
Can we put the URLs for these in now?

Reply:
Not yet - Need to confirm URL structure for patterns